src/Card.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import InputForm from './InputForm'
+import List from './List'
+
+
+const Card = props => {
+  return (
+    <div className="card mt-4"
+          style={{width: 300}}>
+      <div className="card-header">
+        <h3 className="text-center">
+        Todo List 1
+        </h3>
+      </div>
+       <InputForm
+          handleItemInput={props.handleItemInput}
+          newItemSubmitHandler={props.newItemSubmitHandler}
+          value={props.value}
+        />
+
+        <List 
+          list={props.list}
+          handleRemove={props.handleRemove}
+          handleEdit={props.handleEdit}
+          handleCompleted={props.handleCompleted}
+          handleMoveDown={props.handleMoveDown}
+          handleMoveUp={props.handleMoveUp}
+        />
+    </div>
+   
+  )
+}
+export default Card
```

Comment:
One part of Standard is to always have an empty space at the end of the file. 
```suggestion
export default Card

```